esx_service/cli/vmdkops_admin.py

```diff
@@ -811,66 +764,47 @@ def generate_tenant_ls_rows(tenant_list):
     
     return rows
 
-def generate_tuple_from_vm_list(vm_list):
-    """ Generate a list of (vm_uuid, vm_name) pair """
-    if not vm_list:
-        return None, []
-    vms = []
-    for vm_name in vm_list:
-        vm_uuid = vmdk_utils.get_vm_uuid_by_name(vm_name)
-        if not vm_uuid:
-            error_info =  "Cannot find vm_uuid for vm {0}, tenant_create failed".format(vm_name)
-            return error_info, None
-        vms.append((vm_uuid, vm_name))
-    
-    return None, vms
-
 def tenant_create(args):
     """ Handle tenant create command """
-    error_info, vms = generate_tuple_from_vm_list(args.vm_list)
-    if error_info:
-        return operation_fail(error_info)
-      
-    name = args.name 
+    name = args.name
```

Comment:
Issue https://github.com/vmware/docker-volume-vsphere/issues/734 is filed to track this.